packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance/Tests.StaticSSR/TextFieldTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using BlazorWorkspace.Testing.Acceptance;
+using Microsoft.Playwright;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Acceptance.StaticSSR;
+
+public class TextFieldTests : NimbleAcceptanceTestsBase
+{
+    public TextFieldTests(PlaywrightFixture playwrightFixture, NimbleBlazorWebHostServerFixture blazorServerClassFixture)
+        : base(playwrightFixture, blazorServerClassFixture)
+    {
+    }
+
+    [Fact]
+    public async Task TextField_CanRenderAsync()
```

Comment:
Note: With the static SSR render mode, we can't really use component event handlers / call methods / etc, so what this test can do / verify is pretty limited, but it still seemed useful to have one.